src/Symfony/Component/Security/Http/Authentication/AuthenticationSuccessHandlerInterface.php

```diff
@@ -31,5 +31,5 @@ interface AuthenticationSuccessHandlerInterface
      * is called by authentication listeners inheriting from
      * AbstractAuthenticationListener.
      */
-    public function onAuthenticationSuccess(Request $request, TokenInterface $token): Response;
+    public function onAuthenticationSuccess(Request $request, TokenInterface $token): ?Response;
```

Comment:
That would be a BC break, so not something we can do.

Reply:
Widening the return type to allow `null` is fine compiler-wise: https://3v4l.org/rDAPN, so only callers could be impacted. As this method is called by authenticators, I would be ok to do this on 6.3 (with changelog entry).